src/ui/buttons/buttonDesigns/fadeOutLogoShowLabelText.jsx

```diff
@@ -0,0 +1,121 @@
+/* @flow */
+/** @jsx node */
+import { LOGO_CLASS } from '@paypal/sdk-logos/src';
+import { node, Fragment, type ChildType } from 'jsx-pragmatic/src';
+
+import { CLASS } from '../../../constants';
+import { BUTTON_SIZE_STYLE } from '../config';
+
+import type {  LabelOptions, DivideLogoAnimationProps } from './types';
+
+export const FADEOUT_LOGO_SHOW_LABEL_TEXT_CONFIG = {
+    tiny:       { min: BUTTON_SIZE_STYLE.tiny.minWidth },
+    medium:     { max: BUTTON_SIZE_STYLE.medium.maxWidth },
+    cssClasses: {
+        DOM_READY:                  CLASS.DOM_READY,
+        ANIMATION_CONTAINER:        'personalized-design-container',
+        PAYPAL_LOGO:                LOGO_CLASS.LOGO,
+        ANIMATION_LABEL_CONTAINER:  'personalized-label-container',
+        PAYPAL_BUTTON_LABEL:        CLASS.BUTTON_LABEL
+    }
+};
+
+export function FadeoutLogoShowLabelTextComponent({ designLabelText } : LabelOptions) : ChildType {
+    const CONTAINER_CLASS = FADEOUT_LOGO_SHOW_LABEL_TEXT_CONFIG.cssClasses.ANIMATION_CONTAINER;
+    const LABEL_CLASS = FADEOUT_LOGO_SHOW_LABEL_TEXT_CONFIG.cssClasses.ANIMATION_LABEL_CONTAINER;
+   
+    return (
+        <Fragment>
+            <div class={ LABEL_CLASS } data-design-experiment='104530'> <span>{designLabelText}</span></div>
+            <style innerHTML={ `
+                .${ CLASS.DOM_READY } .${ CONTAINER_CLASS } img.${ LOGO_CLASS.LOGO }{
+                    position: relative;
+                }
+                
+                .${ CONTAINER_CLASS } .${ LABEL_CLASS } {
+                    position: fixed;
+                    opacity: 0; 
+                    color: #142C8E;
+                    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+                    font-size: 10px;
+                }
+            ` } />;
+        </Fragment>
+    );
+}
+
+// Returns label container if the button sizes match
+export const getFadeoutLogoShowLabelTextProps = function(document, configuration) : DivideLogoAnimationProps | null {
+    const { ANIMATION_CONTAINER, ANIMATION_LABEL_CONTAINER, PAYPAL_BUTTON_LABEL, PAYPAL_LOGO } = configuration.cssClasses;
+    const { tiny, medium } = configuration;
+    // get the animation main container to force specificity( in css ) and make sure we are running the right animation
+    const animationContainer = (document && document.querySelector(`.${ ANIMATION_CONTAINER }`)) || null;
+    if (!animationContainer) {
+        return null;
+    }
+
+    // return null if animation should not be played for the button size
+    const animationContainerWidth = animationContainer.offsetWidth;
+    if (animationContainerWidth < tiny.min || animationContainerWidth > medium.max) {
+        // remove label element from dom
+        animationContainer.querySelector(`.${ ANIMATION_LABEL_CONTAINER }`).remove();
```

Comment:
Can you add this to the other designs as long as it looks fine in testing?

Reply:
I am not sure what do you mean?